### ISSUE TITLE
Highlight of C++ code doesn't show up properly

### ISSUE THREAD

**Issue by JerryCao1985** [GitbookIO/feedback#322]
## Category

It is a bug. It shows only on gitbook.com, behave normal in Editor.

## What is the current behavior?

The exact step to reproduce the bug needs the source code of my article which I don't know if internal devs have access.
https://jerrycao1985.gitbooks.io/hack-an-algorithm/content/house-robber.html
One can easily notice the first and last code piece have no highlight at all. Although they do in editor.

## What is the desired behavior?

I expect all of the three code pieces have highlight

## Additional information

MacOS Sierra, version 10.12.2
Google Chrome, Version 55.0.2883.95 (64-bit)
GitBook Editor, 6.5.2

Jiayin Cao (Jerry)
https://jerrycao1985.gitbooks.io/hack-an-algorithm/content/house-robber.html

**Comment by Mackiovello** [comment]
Have you tried changing the highlighting plugin from highlight.js to [prism](https://plugins.gitbook.com/plugin/prism)? I had some issues with my C# highlighting before. Changing to prism fixed it.

Just add
```json
{
  "plugins": ["prism", "-highlight"]
}
```

**Comment by Soreine** [comment]
Indeed, you can try to use the `prism` plugin instead, which might be more robuts. But I think it has a bug with the C++ syntax right now (not C however...)

I have seen that 'vector' in this line `vector dp( nums.size() , 0 );` is responsible for the bug in the highlight. You can file an issue on the [highlight.js repository](https://github.com/isagalaev/highlight.js/issues)

**Comment by Soreine** [comment]
The bug in the `prism` plugin should be fixed in 2.0.3
Feel free to give it a try
